docs/exampleinstallation.md

```diff
@@ -0,0 +1,233 @@
+make docker_build
+```
+obtain IMAGE_ID using 
+```
+docker images
+```
+tag and push the image
+
+NOTE: in case of private docker registry, please take a look on our documentation /docs/privateregistry.md
+
+```
+docker tag $IMAGE_ID $REGISTRY_URL/server-controller-prod:latest
+docker push $REGISTRY_URL/server-controller-prod:latest
+```
+### 4.Configure CRD chart and install it 
+```
+vim inference-model-manager/helm-deployment/crd-subchart/values.yaml
+- replace 
+<crd_image_path> with $REGISTRY_URL/server-controller-prod
+<crd_image_tag> with latest 
+<dns_domain_name> with imm.example.com
+cd inference-model-manager/helm-deployment/crd-subchart/
+helm install .
+```
+Check installation
+Issue command
+``` 
+kubectl get pods -n crd
+```
+Expected output:
+```
+NAME                                READY     STATUS             RESTARTS   AGE
+server-controller-c989895b7-pvh55   1/1       Running   0                   17s
+```
+### 5. Install ingress-nginx
+Go to inference-model-manager/helm-deployment/ing-subchart/, open values.yaml file and fill in your environment type (cloud or bare metal).
+Run
+```
+helm install .
+```
+If your environment is bare metal and you want to use Kubernetes Node Port, please take a look on docs/nodeport.md file.
+
+
+### 6. Choose storage provider
+For storing AI models you can choose any S3 compatible provider. If you already have Minio/S3 or other component, Management Api installation guide will show you how to integrate it with our pltform. If not, commands below show how to deploy example Minio component.
+
+```
+cd inference-model-manager/helm-deployment/minio-subchart/
+vi values.yaml #type desired credentials
+helm dep up .
+helm install .
+```
+
+### 7. Install DEX Oauth2Server [dex doc]
+In this step you need to configure DEX connection to identity provider, like LDAP.
+Sample dex configuration for LDAP: https://github.com/dexidp/dex/blob/master/examples/config-ldap.yaml
+Create certificates using inference-model-manager/helm-deployment/dex-subchart/generate-dex-certs.sh and generate-ing-dex-certs.sh scripts. Remember to export DEX_NAMESPACE, DEX_DOMAIN_NAME and ISSUER environment variables, before running those scripts.
+Format of above variables should fit following patterns:
+```
+export ISSUER=https://dex.imm.example.com:443/dex # change 443 port if using kubernetes node port instead of load balancer
+export DEX_NAMESPACE=dex
+export DEX_DOMAIN_NAME=dex.imm.example.com
+```
+Generate certificates:
+```
+cd inference-model-manager/helm-deployment/dex-subchart/certs
+./generate-dex-certs.sh 
+./generate-ing-dex-certs.sh
+```
+
+Following commands deploy Dex and Ldap with our example configuration.
+```
+cd inference-model-manager/tests/deployment/
+helm install --name imm-ldap -f ldap/values.yaml stable/openldap
+```
+After successful deployment (pod is up and running) of ldap run:
+
+```
+cd inference-model-manager/tests/deployment/
+sed -i "s@toreplacedbyissuer@${ISSUER}@g" dex_config.yaml
```

Comment:
Perhaps it would be clearer to use term DEX_DOMAIN_NAME/dex or to to explain what is the issuer. I suspect the path dex might be hardcoded in several places.